src/algebra/lie_algebras.lean

```diff
@@ -0,0 +1,185 @@
+/-
+Copyright (c) 2019 Oliver Nash. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Oliver Nash
+-/
```

Comment:
Just saw the filename... I think in general we use singular. But I'm sure that exceptions exist. I would suggest `src/algebra/lie_algebra.lean` though. See for example your imports on the line below.